docs/atomic-sign.1.md

```diff
@@ -0,0 +1,55 @@
+% ATOMIC(1) Atomic Man Pages
+% Brent Baude
+% August 2016
+# NAME
+atomic-sign- Create a signature for a local image
+# SYNOPSIS
+**atomic sign**
+[**-h**|**--help**]
+
+[**-o**, **--output**]
+[**--sign_key**]
+[ image ... ]
+
+# DESCRIPTION
+**atomic sign** will create a local signature for one or more local images that have 
+been pulled from a registry. Unless overridden, the signature will end up in a directory
+as follows:
+
+ /var/lib/atomic/containers/registry_name/image_name@sha256:manifest/
+
+The signatures are named 'signature-INTEGER' where the integer is increased by 1 each
+time.
+
+The default storage location for signatures is /var/lib/atomic/containers.  A different
+default location can be defined in /etc/atomic.conf with the key **default-sigstore-path**.
```

Comment:
This will need to be expanded for the various more strictly namespaced paths, per `lookaside.go` or similar.